src/main/resources/ERCode/ERCodeEditSheet.xml

```diff
@@ -193,9 +194,13 @@ $xwiki.ssx.use("ERCode.ERCodeEditSheet")
       <code>require(['jquery'], function ($) {
 $(function(){
   var page = $('meta[name="page"]').attr('content'),
-      namespace = $('meta[name="restURL"]').attr('content') + "/objects/",
+      namespace = $('meta[name="restURL"]').attr('content'),
       $detailTable = $('#detailTable')
   ;
+  if(typeof namespace === "undefined"){
+    namespace = $('html').attr('data-xwiki-rest-url');
```

Comment:
It's not a good idea to manually get the attributes from the DOM. If anything changes later, your code will break.

Instead, you can use the [meta AMD module, as described in the JavaScript API documentation](http://platform.xwiki.org/xwiki/bin/view/DevGuide/JavaScriptAPI#HGetsomeinformationaboutthecurrentdocument).